File: demo-ast/samples/java/single_file/SimpleSample.java
/**
 * This is class docstring
 */
public class SimpleSample {
    private int number;
    private String text;

    public SimpleSample(int number, String text) {
        this.number = number;
        this.text = text;
    }

    public void printInfo() {
        System.out.println(text + ": " + number);
    }

    public static void main(String[] args) {
        SimpleSample sample = new SimpleSample(42, "Hello Tree-sitter");
        sample.printInfo();
    }
}

File: demo-ast/samples/python/single_file/simple_sample.py
# import
import os



class Calculator:
    def __int__(self):
        pass

    def sum(self, a, b):
        """
        Sum the two numbers
        Args:
            a: number
            b: number

        Returns: the sum of the two numbers

        """
        return a + b

# Constant
NAME: str = "World"
""" doc..."""

def say_hi(name: str=NAME):
    """
    Docstring ...
    Args:
        name: name

    Returns: None

    """
    print("Hello, " + name)


if __name__ == "__main__":
    calculator = Calculator()
    print(calculator.sum(1, 2))
    say_hi()
